.github/workflows/cibuildwheel.yml

```diff
@@ -52,8 +65,9 @@ jobs:
           CIBW_SKIP: "cp35-* cp36-* cp39-*"
```

Comment:
Might be a little simpler to move to BUILD? `CIBW_BUILD: "cp{37,38}-*"` perhaps? Also, `cp3?-*` won't generalize to 3.10.